content/tracing/languages/go.md

```diff
@@ -48,9 +48,9 @@ ___
 {{% table responsive="true" %}}
 | Framework   | Framework Documentation               | GoDoc Datadog Documentation                                                        |
 |-------------|---------------------------------------|------------------------------------------------------------------------------------|
-| Gin         | https://gin-gonic.github.io/gin/      | https://godoc.org/github.com/DataDog/dd-trace-go/tracer/contrib/gin-gonic/gintrace |
-| Gorilla Mux | http://www.gorillatoolkit.org/pkg/mux | https://godoc.org/github.com/DataDog/dd-trace-go/tracer/contrib/gorilla/muxtrace   |
-| gRPC        | https://github.com/grpc/grpc-go       | https://godoc.org/github.com/DataDog/dd-trace-go/tracer/contrib/tracegrpc          |
+| Gin         | https://gin-gonic.github.io/gin/      | https://godoc.org/github.com/DataDog/dd-trace-go/contrib/gin-gonic/gintrace |
```

Comment:
This is not correct. Note that the new paths are also different. They match `user/repo` as it is on `github.com/user/repo`. So in this case it's: `https://godoc.org/github.com/DataDog/dd-trace-go/contrib/gin-gonic/gin`